src/components/nature-of-business/NatureOfBusiness.vue

```diff
@@ -285,7 +287,12 @@ export default class NatureOfBusiness extends Vue {
   padding: 8px;
 }
 
+#nob-change-btn {
+  margin-right: -14px;
```

Comment:
This is so the button label is in line with the right margin, but when highlight the padding is the same on the left and right (it goes past the margin), ie:

![image](https://user-images.githubusercontent.com/10002889/170580624-dc94e114-a37a-4ea1-aac8-c91105fe31e0.png)

Reply:
I think* the Edit UI might already have some compensation for this, so might need an update there, super minor 👍

Reply:
It does. I'm in there already so I removed the v-deep.